sopel/modules/help.py

```diff
@@ -82,21 +82,20 @@ def help(bot, trigger):
 
 def create_list(bot, msg):
     msg = 'Command listing for {}@{}\n\n'.format(bot.nick, bot.config.core.host) + msg
-    payload = {"content": msg}
-    headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
 
     try:
-        result = requests.post('https://ptpb.pw/', json=payload, headers=headers)
+        result = requests.post('https://clbin.com/', data={'clbin':msg})
```

Comment:
This line introduces a code-style error.
```suggestion
        result = requests.post('https://clbin.com/', data={'clbin': msg})
```